package.json

```diff
@@ -11,8 +11,10 @@
     "url": "https://github.com/JedWatson/react-select.git"
   },
   "dependencies": {
+    "create-react-class": "15.5.1",
     "classnames": "^2.2.4",
-    "react-input-autosize": "^1.1.0"
+    "react-input-autosize": "^1.1.0",
```

Comment:
This dependency will need to be updated as well. I've created a PR [here](https://github.com/JedWatson/react-input-autosize/pull/79) to do so